psibomana/react-redux-project
# Tests fix

**What does this PR do?**
Fix of failing test cases and code refactoring

**Description of Task to be completed?**
Updated failing test cases and restructured test files

**How should this be manually tested?**
Clone and set up the project using the instruction in the README.
Use `$ git checkout tests_fix` to check out the branch containing the changes of this pull request
Use `$ yarn test` to run the application tests
All the tests should run successfully.
